### PR TITLE
Update get-allshops-repository

### PR DESCRIPTION
**Pull request by bavanya**

### CODE REVIEW

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java [hunk 1]
```diff
@@ -59,7 +60,19 @@ public UUID createShop(CreateShopRequest createShopRequest) {
...
+		List<Shop> shopList = new ArrayList<>();
+
+		for (Map.Entry<UUID, Shop> mapOfShop : shopMap.entrySet()) {
+			UUID mapKey = mapOfShop.getKey();
+			Shop mapValue = mapOfShop.getValue();
+
+			if (mapValue.getStatus() == ShopStatus.ACTIVE) {
+				shopList.add(mapValue);
+			}
+		}
```
Comment (priytanvani) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java:74]: Let's try to use java streamz for making our code more readable here.

Comment (bavanya) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java]: Yes done

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java [hunk 2]
```diff
@@ -59,7 +61,9 @@ public UUID createShop(CreateShopRequest createShopRequest) {
 		return newShop.getShopId();
 	}
 
-	public Map<UUID, Shop> getAllShops() {
-		return shopMap;
+	public List<Shop> getAllShops() {
+
```
Comment (priytanvani) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java:65]: Nit. remove extra line.

Comment (bavanya) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java]: Done

**Pull request merged**